### DISCUSSION TITLE
어떻게 하면 플레이어가 움직이는 플랫폼과 함께 움직일까요?

### DISCUSSION THREAD

**Discussion by ted002** [naverz/zepeto-studio-kor#402] [5. Q&A]
https://user-images.githubusercontent.com/73985764/154424019-133dc62e-ac55-41db-86f4-89c817d2cd6c.mp4

영상에서 나온것처럼 현재 이동하는 플랫폼을 만들었는데, 이 플랫폼에 리지드 바디도 넣어보고 새로운 피직스 마테리얼도 넣어봤는데도 플레이어가 플랫폼이 이동만하면 함께 이동하지 않고 떨어지네요. 

플레이어에 리지드 바디를 추가로 넣어보기도했는데, 여전히 실패해서 물어봅니다. 

뭔가 좋은 방법 없을까요?

**Comment by x4cf** [comment]
플레이어에서 아래방향으로 레이케스트를 쏴서 아래의 땅이 움직이면 같이 움직이거나,
땅에서 위에 플레이어를 감지해서 플레이어를 같이 이동시키거나 하는 방법이 있습니다

**Reply by x4cf** [reply]
이곳에서 마스터 타이머와 무빙큐브를 다운로드 받아 사용하는 방법도 있어요 :D
https://cheesafizz.notion.site/f4b997d8384347ff9922fc770bc2d864

**Reply by ted002** [reply]
감사합니다 한번 해보겠습니다

**Reply by LeeYongSu1** [reply]
땅에 닿으면 땅 오브젝트의 자식으로 플레이어의 오브젝트를 넣으면 됩니다
